macOS/DuckDuckGo/MainWindow/MainViewController.swift

```diff
@@ -164,7 +170,9 @@ final class MainViewController: NSViewController {
         super.viewDidAppear()
         mainView.setMouseAboveWebViewTrackingAreaEnabled(true)
         registerForBookmarkBarPromptNotifications()
+
         adjustFirstResponder(force: true)
+        tryToShowSetAsDefaultAndAddtoDockIfNeeded()
```

Comment:
Maybe botho `try` and `ifNeeded` aren’t needed at the same time? :) How about `showSetAsDefaultAndAddToDockIfNeeded`? (Mind the capital `T` in `AddToDock`).